src/geometry/LineUtil.js

```diff
@@ -240,3 +241,59 @@ export function _flat(latlngs) {
 	console.warn('Deprecated use of _flat, please use L.LineUtil.isFlat instead.');
 	return isFlat(latlngs);
 }
+
+/* @function polylineCenter(latlngs: LatLng[], map: Map): LatLng
+ * Returns the center of the `Polyline` of the passed LatLngs.
+ * The returned LatLng will be on the `Polyline`.
+ */
+export function polylineCenter(latlngs, map) {

```

Comment:
Not sure that we are ready to expose that functions, without any discussion.
- As it is (relatively) low-level function, what if we pass `crs` and `zoom` directly, instead of map?
- What if we give reasonable defaults for `crs` and `zoom`?

Reply:
I think this is a good idea